tests/integration/helpers/cluster.py

```diff
@@ -802,13 +801,13 @@ def start(self, destroy_dirs=True):
             subprocess_check_call(clickhouse_start_cmd)
             print("ClickHouse instance created")
 
-            start_deadline = time.time() + 20.0  # seconds
+            start_timeout = 20.0  # seconds
             for instance in self.instances.values():
                 instance.docker_client = self.docker_client
                 instance.ip_address = self.get_instance_ip(instance.name)
 
                 print("Waiting for ClickHouse start...")
-                instance.wait_for_start(start_deadline)
+                instance.wait_for_start(start_timeout)
```

Comment:
BTW it is not the same (deadline was common, now it is separate). But I suppose it is ok